src/scripts/interns.js

```diff
@@ -59,9 +73,31 @@ export async function displayInternWeekTable() {
     const row = document.createElement("tr"); //creating group row
 
     const groupNum = document.createElement("td"); //Group num column
-    groupNum.textContent = "Group " + (index + 1);
+    groupNum.innerHTML = `<p>Group ${index + 1}</p>`;
+
+    const addBtn = document.createElement("button"); //add edit button
+    addBtn.className = "edit";
+    addBtn.id = "add-intern";
+    addBtn.type = "button";
+    addBtn.innerHTML = `<i class="fa-solid fa-user-plus"></i>`;
+    groupNum.appendChild(addBtn);
+
+    displayAddModal(addBtn, pair, index); //display add functionality
+
+    const rmBtn = document.createElement("button"); //remove edit button
+    rmBtn.className = "edit";
+    rmBtn.id = "remove-intern";
+    rmBtn.type = "button";
+    rmBtn.innerHTML = `<i class="fa-solid fa-user-minus"></i>`;
+    groupNum.appendChild(rmBtn);
     row.appendChild(groupNum);
 
+    displayRemoveModal(rmBtn, pair, index);
+
+    if (pair.length === 0) {
+      rmBtn.innerHTML = `<i class="fa-solid fa-minus"></i>`;
+    }
+
```

Comment:
I would rename this button to something more clear

```suggestion
    const addInternToPairButton = document.createElement("button"); //add edit button
    addInternToPairButton.className = "edit";
    addInternToPairButton.id = "add-intern";
    addInternToPairButton.type = "button";
    addInternToPairButton.innerHTML = `<i class="fa-solid fa-user-plus"></i>`;
    groupNum.appendChild(addInternToPairButton);

    displayAddModal(addInternToPairButton, pair, index); //display add functionality

    const removeInternFromPairButton = document.createElement("button"); //remove edit button
    removeInternFromPairButton.className = "edit";
    removeInternFromPairButton.id = "remove-intern";
    removeInternFromPairButton.type = "button";
    removeInternFromPairButton.innerHTML = `<i class="fa-solid fa-user-minus"></i>`;
    groupNum.appendChild(removeInternFromPairButton);
    row.appendChild(groupNum);

    displayRemoveModal(removeInternFromPairButton, pair, index);

    if (pair.length === 0) {
      removeInternFromPairButton.innerHTML = `<i class="fa-solid fa-minus"></i>`;
    }

```

Reply:
Done!